src/Nethermind/Nethermind.TxPool/TxFilteringState.cs

```diff
@@ -18,5 +17,17 @@ public TxFilteringState(Transaction tx, IAccountStateProvider accounts)
     }
 
     private AccountStruct? _senderAccount = null;
```

Comment:
Does this need to be nullable? Could check if default (both hashes are 0)

Having it nullable goes from 128 bytes -> 160 bytes (32 byte gain)
```
Type layout for 'AccountStruct'
Size: 128 bytes. Paddings: 0 bytes (%0 of empty space)
|====================================================|
|  0-31: UInt256 <Nonce>k__BackingField (32 bytes)   |
|----------------------------------------------------|
| 32-63: UInt256 <Balance>k__BackingField (32 bytes) |
|----------------------------------------------------|
| 64-95: ValueHash256 StorageRoot (32 bytes)         |
|----------------------------------------------------|
| 96-127: ValueHash256 CodeHash (32 bytes)           |
|====================================================|
```
```
Type layout for 'Nullable`1'
Size: 160 bytes. Paddings: 0 bytes (%0 of empty space)
||
||
```

Reply:
Is it better now?